TweetMate/src/main/java/com/ybsystem/tweetmate/activities/TimelineActivity.java

```diff
@@ -0,0 +1,115 @@
+package com.ybsystem.tweetmate.activities;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.ybsystem.tweetmate.fragments.timeline.DetailTimeline;
+import com.ybsystem.tweetmate.fragments.timeline.MainTimeline;
+import com.ybsystem.tweetmate.fragments.timeline.TalkTimeline;
+import com.ybsystem.tweetmate.fragments.timeline.UserTimeline;
+import com.ybsystem.tweetmate.models.entities.Column;
+import com.ybsystem.tweetmate.models.entities.twitter.TwitterStatus;
+import com.ybsystem.tweetmate.models.entities.twitter.TwitterUser;
+import com.ybsystem.tweetmate.models.enums.ColumnType;
+import com.ybsystem.tweetmate.databases.PrefAppearance;
+
+import static com.ybsystem.tweetmate.models.enums.ColumnType.*;
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class TimelineActivity extends ActivityBase {
+
+    private ColumnType mColumnType;
+    private TwitterStatus mStatus;
+    private TwitterUser mUser;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Get intent
+        if (savedInstanceState == null) {
+            mColumnType = (ColumnType) getIntent().getSerializableExtra("COLUMN_TYPE");
+            mStatus = (TwitterStatus) getIntent().getSerializableExtra("STATUS");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "STATUS" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1192)